pkg/rancher-components/src/components/Pill/RcTag/RcTag.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { RcTagProps } from './types';
+
+const props = withDefaults(defineProps<RcTagProps>(), { disabled: false, showClose: false });
+const emit = defineEmits(['close']);
+</script>
+
+<template>
+  <div
+    class="rc-tag"
+    :class="{[props.type]: true, disabled: props.disabled}"
+  >
+    <slot name="default" />
+    <button
+      v-if="props.showClose"
+      :aria-label="props.closeAriaLabel"
+      @click="emit('close')"
+    >
+      <i class="icon icon-close" />
+    </button>
```

Comment:
This button is missing focus styles when it receives keyboard focus. I suggest that we use `RcButton` so that we can get consistent button behavior out of the box. The ghost role should be able to satisfy the need for having a barebones button that we can style ourselves.

Reply:
Done